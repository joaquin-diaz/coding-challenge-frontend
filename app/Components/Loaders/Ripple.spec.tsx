import * as React from 'react';

import Ripple from './Ripple';
import { mount } from 'enzyme';

describe('<Ripple />', () => {
  it('should render', () => {
    const wrapper = mount(<Ripple color="black" />);
    expect(wrapper).toMatchSnapshot();
  });
});
