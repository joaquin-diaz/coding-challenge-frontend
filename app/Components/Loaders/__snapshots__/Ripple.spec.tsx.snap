// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ripple /> should render 1`] = `
.emotion-0 {
  display: inline-block;
  position: relative;
  width: 64px;
  height: 64px;
}

.emotion-0 div {
  position: absolute;
  border: 4px solid black;
  opacity: 1;
  border-radius: 50%;
  -webkit-animation: lds-ripple 1s cubic-bezier(0,0.2,0.8,1) infinite;
  animation: lds-ripple 1s cubic-bezier(0,0.2,0.8,1) infinite;
}

.emotion-0 div:nth-child(2) {
  -webkit-animation-delay: -0.5s;
  animation-delay: -0.5s;
}

<Ripple
  color="black"
>
  <Styled(div)
    color="black"
  >
    <div
      className="emotion-0"
      color="black"
    >
      <div />
      <div />
    </div>
  </Styled(div)>
</Ripple>
`;
